models/model1_part2.ipynb

```diff
@@ -0,0 +1,611 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    # only 1000 additional samples, this is done to address imbalanced classes
</pre><p>This comment isn't clear. What are you trying to do here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PanwarJayant/spot/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>